hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -1452,6 +1452,20 @@ public String getCreateTableString() {
 	}
 
 	/**
+	 * Command used to alter a table.
+	 *
+	 * @return The command used to alter a table.
+	 * @since 5.2.11
+	 */
+	public String getAlterTableString() {
+		final StringBuilder sb = new StringBuilder( "alter table" );
+		if ( supportsIfExistsAfterAlterTable() ) {
+			sb.append( " if exists" );
+		}
+		return sb.toString();
+	}
+
+	/**
```

Comment:
Yeah, why not - will probably reduce the code duplication even further where those statements are built.